### DISCUSSION TITLE
stuck on a orange pi 5/ghost process killer work around?

### DISCUSSION THREAD

**Discussion by develperbayman** [termux/termux-app#3587] [Q&A] [answered]
yeah im stuck on a orange pi 5 so upgrading android with the developer setting to stop the process killer isnt a option  is there a workaround i missed sorry im sure its out there but theres alot to go thru any suggestion would be awesome

**Comment by ChiefMikeK** [comment]
:shipit: It normally helps if you specify what version from-to of andriod you're tring to update 🧝‍♀️

**Reply by develperbayman** [reply]
sorry my bad basically all opi5 is stuck on android 12 before the developer setting was implemented to fix the ghost killer

**Comment by twaik** [comment, marked as answer]
**Phantom Process Killing** 📱   🔫 👻 
Termux being randomly killed and displaying `[Process completed (signal 9) - press Enter]` is a known problem. It is caused by an Android battery optimization called the phantom process killer.
- More information about the issue:
<https://github.com/termux/termux-app/issues/2366>
- Potential solutions
<https://github.com/agnostic-apollo/Android-Docs/blob/master/en/docs/apps/processes/phantom-cached-and-empty-processes.md#commands-to-disable-phantom-process-killing-and-tldr> (link above takes you to end chapter; read the full thread for an in-depth description of what's happening)
